infra.bs

```diff
@@ -216,8 +217,11 @@ in parentheses. [[!UNICODE]]
 
```

Comment:
Should we mention that these definitions come from Unicode? It seems like we should give them credit for this edifice in some way.